### ISSUE TITLE
Way to match the CSS webkit filters?

### ISSUE THREAD

**Issue by saurabhudaniya200** [meltingice/CamanJS#214]
I am using CamanJS in my project and it is working fine as expected, but in my application I have a whole bunch of images which I put inside the container and then apply the CSS filters directly to the container instead of applying the effects to all the images individually, the CSS filters I use are below

```
-webkit-filter: brightness(100%) contrast(100%) saturate(100%) hue-rotate(0deg) sepia(0.7) grayscale(1);
}
```

Once user hits next then I use the Caman JS to create the actual images, so is there an algorithm that I can use to match the values especially brightness?

**Comment by AshwinTayson** [comment]
Yes. This has been asked in other forums also. I hope we get answer from the author soon.
